include/modules/sv_detection_methods/analyze_split_read_method.hpp

```diff
@@ -36,6 +36,32 @@ void split_string(std::string const & str, Container & cont, char const delim =
  */
 void retrieve_aligned_segments(std::string const & sa_string, std::vector<AlignedSegment> & aligned_segments);
 
+/*! \brief Calculates the end respectively start position of two consecutive mates (current and next
+ *         [AlignedSegment](\ref AlignedSegment)) depending on their orientation and corrects the position of the second
+ *         mate, if they are overlapping.
+ *
+ * \param[in] current          - current [AlignedSegment](\ref AlignedSegment)
+ * \param[in] next             - consecutive next [AlignedSegment](\ref AlignedSegment)
+ * \param[in] distance_on_read - distance between the consecutive aligned_segments on the read
```

Comment:
Use `AlignedSegments` as above or
```suggestion
 * \param[in] distance_on_read - distance between the consecutive aligned segments on the read
```